src/ddmd/declaration.d

```diff
@@ -265,7 +265,7 @@ extern (C++) abstract class Declaration : Dsymbol
         return (storage_class & STCfinal) != 0;
     }
 
-    final bool isAbstract()
+    bool isAbstract()
```

Comment:
Need to mark this as virtual in C++ headers.

Reply:
done